sources/Renderer/TextureUtils.h

```diff
@@ -64,17 +64,19 @@ LLGL_EXPORT SubresourceCPUMappingLayout CalcSubresourceCPUMappingLayout(
     const Extent3D&     extent,
     std::uint32_t       numArrayLayers,
     const ImageFormat   imageFormat,
-    const DataType      imageDataType
+    const DataType      imageDataType,
+    uint32_t            srcDataStride = 0
```

Comment:
I once thought about changing all the `std::uint32_t` types back to just plain C `uint32_t` but I originally wanted the code base to be explicitly C++ and now I'm stuck with it. So I'd prefer to stick with the one that explicitly refers to C++'s `std` namespace.